docs/Encyclopedia/pages/TCGA_VCF_1.1v2.md

```diff
@@ -537,13 +537,13 @@ are optional.
+|           |            |                                                                            |                                                                                                                                                                                                                                                                                                                              | *([1-22], X, Y, MT, <ID\>)*                                                             | <ctg1>              |                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | 2         | POS        | Yes                                                                        | *Position*: The reference position, with the 1st base having position 1                                                                                                                                                                                                                                                      | Non-negative integer                                                                     | 1110696               | Yes            | \---                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |

```

Comment:
this pre-existing escape is curious: `\---`.  is that to prevent the dashes from being interpreted as the header marker?

Reply:
That was probably the idea though it doesn't seem like its actually necessary.